api/app/src/routes/medical/organization.ts

```diff
@@ -35,6 +37,10 @@ router.post(
     const createOrg: OrganizationCreateCmd = { cxId, ...data };
     const org = await createOrganization(createOrg);
 
+    // temp solution until we migrate to FHIR
+    const fhirOrg = toFHIR(org);
+    await upsertOrgToFHIRServer(fhirOrg);
```

Comment:
Just relating to the comments on error handling of these: if we want to abstract away errors to FHIR server from our users, we could make these asynchronous. 

Here and other similar places.

Moot comment if we want to expose FHIR server related errors.